tools/mo/unit_tests/mo/utils/freeze_placeholder_test.py

```diff
@@ -61,8 +60,7 @@ def get_test_default_frontends():
     return {"onnx": "new", "tf": "legacy"}
 
 
-@generator
-class TestMoFreezePlaceholder(unittest.TestCase):
+class TestMoFreezePlaceholder():
     def setUp(self):
```

Comment:
probably `setUp` is not longer working now that is why the input model is not found. Please fix